andela/mervels-bn-backend
# #168131405 Setup unit testing. Ensure all implemented functionality is tested henceforth.

#### What does this PR do?
- it sets up unit tests for implemented tasks
#### Description of Task to be completed?
- there is a test folder inside src directory.
- that folder is dedicated for all test files.
##### dependencies
- mocha
- chai
- chai-http
#### How should this be manually tested?
-clone the repo
- run npm install
- write the tests for a feature you are about to implement
- put your tests into any .js file under the ./src/test directory 
- implement the feature to match the tests
- run npm test
- you should see a list of passing or failing tests inclusively.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131405](https://www.pivotaltracker.com/story/show/168131405)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
